events/MTG_USA.txt

```diff
@@ -4396,36 +4396,36 @@ country_event = {
 		hidden_effect = { JAP = { country_event = { id = panay_incident.3 days = 2 } } } #Delay added so events are spaced out - historically it took months to get JAP to agree to compensation, but that'd be a bit hard to follow in-game, so timelines compressed.
 		add_opinion_modifier = { target = JAP modifier = panay_inc_opinion_modifier }
 	}
-	option = { #War
```

Comment:
It will work just fine by removing the option. But I was talking with voigt and he raised a valid issue that USA going for an interventionist path should probably be allowed to declare this war. They're hit with heavy penalties for it anyway.

I would suggest you tried to change it so that the option is not removed, but only visible when USA is not on undisturbed isolation. You can do that by adding `trigger` block in the `option` block for this event.

Reply:
Let me get the trigger in and test.  the alex/master should be the origin and my fork upstream correct?